client/components/Help/HelpPortalModal/HelpPortalModal.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect, useCallback, FC } from 'react'
+import { Modal, ModalHeader, ModalBody } from 'reactstrap'
+
+type Props = {
+  el: HTMLElement | null
+}
+
+const HelpPortalModal: FC<Props> = ({ el = null }) => {
+  const [show, setShow] = useState(false)
+
+  const handleOpen = useCallback(() => {
+    setShow(true)
+  }, [])
+
+  const handleClose = useCallback(() => {
+    setShow(false)
+  }, [])
+
+  useEffect(() => {
+    const e: any = el || "a[data-target='#help-portal']"
+    $(e).on('click', handleOpen)
```

Comment:
If you use button component on Storybook and remove props, You can write:
```suggestion
    $("[data-target='#help-portal']").on('click', handleOpen)
```

Reply:
Since the element of `[data-target='#help-portal']` is dynamically generated (in the case of storybook), it is supported by changing to `$(document).on`.